paint-prototype/Assets/Fork/Tom/Scripts/PlayerCamera.cs

```diff
@@ -0,0 +1,45 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class PlayerCamera : MonoBehaviour
+{
+    [SerializeField] private float sensitivityX;
+    [SerializeField] private float sensitivityY;
+
+    Camera cam;
```

Comment:
Why `cam` and not `camera`? If the variable will be part of the whole class it should have more explicit names.